geode-core/src/main/java/org/apache/geode/cache/client/internal/pooling/PooledConnection.java

```diff
@@ -37,7 +37,7 @@
  * @since GemFire 5.7
  *
  */
-class PooledConnection implements Connection {
+public class PooledConnection implements Connection {
```

Comment:
Fix looks good to me.
But here just wanted to confirm that the visibility of the class was made public so that it can be mocked in a test class ? is that correct?

Reply:
That is correct.  The reason for this is the ConnectionManagerImpl asserts that the Connection is a PooledConnection, and we are encountering that assert in our test.  See `ConnectionManagerImpl.returnConnection()` and `ConnectionManagerImpl.exchangeConnection()`